frontend/src/pages/home/OrganizeSection.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import {
+  Bullseye,
+  CardBody,
+  Gallery,
+  PageSection,
+  PageSectionProps,
+  Stack,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import projectIcon from '~/images/UI_icon-Red_Hat-Folder-Color.svg';
+import pipelineIcon from '~/images/UI_icon-Red_Hat-Branch-Color.svg';
+import chartIcon from '~/images/UI_icon-Red_Hat-Chart-Color.svg';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import useServingPlatformStatuses from '~/pages/modelServing/useServingPlatformStatuses';
+import TypeBorderedCard from '~/concepts/design/TypeBorderedCard';
+import { SectionType } from '~/concepts/design/utils';
+import OrganizeGallery from '~/pages/home/OrganizeGallery';
+import PipelinesGallery from '~/pages/home/PipelinesGallery';
+import ServingGallery from '~/pages/home/ServingGallery';
+
+interface OrganizeCardProps {
+  title: string;
+  imgSrc: string;
+  sectionType: SectionType;
+  selected: boolean;
+  onSelect: () => void;
+}
+
+const OrganizeCard: React.FC<OrganizeCardProps> = ({
+  title,
+  imgSrc,
+  sectionType,
+  selected,
+  onSelect,
+}) => (
+  <TypeBorderedCard
+    sectionType={sectionType}
+    selectable
+    selected={selected}
+    onClick={() => onSelect()}
+    onKeyDown={(event) => {
+      if (event.key === 'Enter' || event.key === ' ') {
+        event.preventDefault();
+        event.stopPropagation();
+        onSelect();
+      }
+    }}
+    tabIndex={0}
+  >
+    <CardBody>
+      <Stack hasGutter>
+        <Bullseye>
+          <img height={32} src={imgSrc} alt="" />
+        </Bullseye>
+        <Bullseye>{title}</Bullseye>
+      </Stack>
+    </CardBody>
+  </TypeBorderedCard>
+);
+
+type OrganizeSectionProps = PageSectionProps & {
+  allowCreateProjects: boolean;
+};
+
+const OrganizeSection: React.FC<OrganizeSectionProps> = ({ allowCreateProjects, ...rest }) => {
+  const { pipelinesServer } = usePipelinesAPI();
```

Comment:
API is not available outside a project context... I think you mean to check the `useIsAreaAvailable` for Pipelines.